docs/src/navigation/sectionalTabs/sectionalTabsSample.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
Please make sure if this is needed or not in 9.15.3, if not please remove.

Reply:
not needed, cherry picked by error.

Reply:
So, this file will be removed from this PR?

Reply:
of course, it was removed

Reply:
Confirmed removal in 53d7c67.
I'll keep thread open for @morethanfire 's benefit.